poll/check.go

```diff
@@ -0,0 +1,38 @@
+package poll
+
+import (
+	"net"
+	"os"
+)
+
+// Check is a function which will be used as check for the WaitOn method.
+type Check func(t LogT) Result
+
+// FileCheck looks on filesystem and check that the path exists.
+func FileCheck(path string) Check {
+	return func(t LogT) Result {
+		_, err := os.Stat(path)
+		if err != nil && os.IsNotExist(err) {
```

Comment:
`os.IsNotExist()` already handles `nil` (https://play.golang.org/p/FiZD0nru0Zt), so no need to check it here. This can be `if os.IsNotExist(err)`.

Reply:
I didn't know about this one. Thanks!